blackbox/docs/sql/administration/user_management.txt

```diff
@@ -0,0 +1,93 @@
+.. _administration_user_management:
+
+User Management
+===============
+
+.. note::
+
+  User Management is only part of the CrateDB Enterprise Edition.
+
+A CrateDB cluster contains a set of database users. A database user is a
+principle at cluster level.
+
+Users can be created using the `CREATE USER`_ statement and removed using the
+analoguous `DROP USER`_ statement. These statements are database management
+statements that can only be invoked by existing superusers that already exist
```

Comment:
These statements are database management statements that can only be invoked by the superuser `crate`.